libs/template/virtual-view/src/lib/resize-observer.ts

```diff
@@ -0,0 +1,42 @@
+import { DestroyRef, inject, Injectable } from '@angular/core';
+import { Observable, ReplaySubject, Subject } from 'rxjs';
+import { distinctUntilChanged, tap } from 'rxjs/operators';
+
+/**
+ * A service that observes the resize of the elements.
+ *
+ * @developerPreview
+ */
+@Injectable()
+export class RxaResizeObserver {
```

Comment:
Maybe also should go to cdk

Reply:
this one should def. go to CDK as it's also used in the virtual-scroll package